### PR TITLE
Adding Unencrypted F5 Cookie disclosure

### PR DESCRIPTION
**Pull request by PR3R00T**
Locate F5 Load balancer Persistence cookies, use other tools to decode these cookies to disclose internal IP address and port number of the service.

### CODE REVIEW

File: tokens/unencrypted-bigip-ltm-cookie.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+requests:
+  - method: GET
+    path:
+      - "{{BaseURL}}"
+    redirects: true
+    matchers:
+      - type: regex
+        regex:
+          - '(BIGipServer[a-z\_\.\-\~0-9A-Z]*)=([0-9a-zA-Z\.]*;)'
```
Comment (dwisiswant0) [inline tokens/unencrypted-bigip-ltm-cookie.yaml:16]: After I tried using the regEx patterns that you used, it didn't match for:
```http
Set-Cookie: BIGipServerPOOLnAmE=127.0.0.1.1337.0000; path=/`
```

Comment (PR3R00T) [reply tokens/unencrypted-bigip-ltm-cookie.yaml:16]: Hey!
Thanks for your feedback!
Perhaps I'm doing something wrong but a quick check and it appears the regEx is right? 

image

That being said, I do believe regEx to be a direct relative to Hades himself.

**Comment by PR3R00T** [comment]
@bauthard @dwisiswant0  any update?

**Review by dwisiswant0 — APPROVED** [review]

**Comment by bauthard** [comment]
Hi @PR3R00T, 

Thank you for the PR, Can you please list some reference or article that makes use of issue?

**Comment by PR3R00T** [comment]
Report: https://hackerone.com/reports/330716

**Comment by PR3R00T** [comment]
Article: https://www.intelisecure.com/how-to-decode-big-ip-f5-persistence-cookie-values/
Fix: https://support.f5.com/csp/article/K23254150 
Report: https://hackerone.com/reports/330716

:)

**Comment by bauthard** [comment]
Thank you @PR3R00T for adding references and sharing this template.

**Pull request merged**